tools/generator/Tests/expected/InterfaceMethodsConflict/InterfaceMethodsConflict.xml

```diff
@@ -28,6 +28,37 @@
 		<method abstract="false" deprecated="not deprecated" final="false" name="irrelevant" native="false" return="void" static="false" synchronized="false" visibility="public">
 		</method>
 	  </class>
+	  <class abstract="false" deprecated="not deprecated" extends="java.lang.Object" extends-generic-aware="java.lang.Object"
+	        final="false" name="FieldMethodNameColisionClass" static="false" visibility="public">
```

Comment:
We should probably add a `setFoo()` method for good measure -- a setter w/o a corresponding getter.

Reply:
I did, I called it setAge

Reply:
Sorry, I thought I saw a `getAge()`... :-/